src/App.css

```diff
@@ -71,3 +110,153 @@ footer nav img {
   max-height: 20px;
   max-width: 20px;
 }
+
+/* Functional styles */
+
```

Comment:
I am! Technically this _is_ still regular css. Being able to describe what you want using classes is great. Combining it with react components to give you both flexibility and abstraction. 

Honestly, just needed to choose a scalable approach and chose this way of organizing the css. It was fun.

Reply:
This is very similar to how [Tailwind CSS](https://tailwindcss.com/docs/utility-first) works. I think it would be a really interesting side-quest to explore bringing it into this project if you wanted to go down the "utility styles as classnames" approach, to keep it consistent with an industry standard. For a small project, it's fine to make a few small utility functions, but over time this file will become really big (every time you need a new CSS state) and could be hard to explore without documentation.